cypress/lib/helper.js

```diff
@@ -51,13 +51,19 @@ export function checkBookingsOrder(expectedData) {
 export function createBookingsDataTest() {
```

Comment:
what's the difference between this function and seed bookings?

Reply:
there are no differences, but we tried in a previous task to use the data from there, but we didn't succeed